build.sbt

```diff
@@ -10,12 +10,12 @@ organization := "com.dwijnand"
      scmInfo := Some(ScmInfo(url("https://github.com/dwijnand/sbt-dynver"), "scm:git:git@github.com:dwijnand/sbt-dynver.git"))
 
        sbtPlugin           := true
-      sbtVersion in Global := "1.0.0" // must be Global, otherwise ^^ won't change anything
-crossSbtVersions           := List("0.13.17", "1.0.0")
+      sbtVersion in Global := "1.1.5" // must be Global, otherwise ^^ won't change anything
+crossSbtVersions           := List("0.13.17", "1.1.5")
```

Comment:
The use of 1.0.0 is intentional here (and the line above) as it's the minimum version this plugin needs.